extensions/blocks/map/edit.js

```diff
@@ -269,12 +279,18 @@ class MapEdit extends Component {
 				</div>
 			</Fragment>
 		);
+		const placholderPreview = (
+			<div>
+				<img alt="Map Preview" src={ previewPlaceholder } />
```

Comment:
I think it would be nice if this were translated.

Reply:
Fixed